src/test/java/net/bhardy/bizzo/billing/PolicyBuilderTest.java

```diff
@@ -43,6 +53,41 @@ public void testBuilder() {
 
         LocalDate oneSaturday = LocalDate.of(2017, 8, 5);
         assertFalse(policy.isDueOn(oneSaturday));
+        LocalDate maybeSunday = LocalDate.of(2017, 8, 6);
+        assertFalse(policy.isDueOn(maybeSunday));
+        LocalDate shouldBeMonday = LocalDate.of(2017, 8, 7);
+        assertTrue(policy.isDueOn(shouldBeMonday));
+    }
+
+    @Test
+    public void upcomingDueDates_monthlySimple() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)